src/lightning_lite/plugins/precision/precision.py

```diff
@@ -67,7 +67,7 @@ def optimizer_step(
         """Hook to run the optimizer step."""
         return optimizer.step(**kwargs)
 
-    def get_main_params(self, optimizer: Optimizer) -> _PARAMETERS:
+    def main_params(self, optimizer: Optimizer) -> _PARAMETERS:
```

Comment:
Don't like the name, but reverting it to avoid breaking change (although this only exists for apex which nobody uses).